modulemd/tests/test_data/upgrades/packager_v3_to_stream_v3.yaml

```diff
@@ -152,4 +167,11 @@ data:
                 ref: somecoolbranchname
                 buildorder: 100
 ...
-
+---
```

Comment:
What is the trigger for this modulemd-defaults document appearing? Did you intend to include `data.profiles.minimal.default: true` in `modulemd_packager_v3.yaml`?

Reply:
Yes, did I miss that?

Reply:
Fixed.